dev-client/src/components/home/repositionCamera.ts

```diff
@@ -0,0 +1,31 @@
+import {CameraRef} from '@rnmapbox/maps/lib/typescript/components/Camera';
+
+export const repositionCamera = (
+  feature: GeoJSON.Feature,
+  zoomLevel: number,
+  animationDuration: number,
+  paddingBottom: number,
+  cameraRef?: React.RefObject<CameraRef>,
+) => {
```

Comment:
nit: this could probably benefit from named parameters at this point, which would also make it easier to provide defaults for e.g. paddingBottom!